PublicationsRouter/v2/api/API.md

```diff
@@ -74,76 +74,94 @@ If you have publicly hosted content (e.g. splash pages, full-text web pages, or
+        Content-Type: multipart/form-data; boundary=FulltextBoundary
     
-    Content-Disposition: form-data; name="content"
-    Content-Type: application/zip
-    
-    [Package]
-    
-    --FulltextBoundary--
+    Body:
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="metadata"
+        Content-Type: application/json
+
+        [Incoming Notification JSON]
+
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="content"
+        Content-Type: application/zip
+
+        [Package]
 
-If you are carrying out this request you MUST include the **content.packaging_format** field in the notification metadata and populate it with the appropriate format identifier as per the [Packaging Format](https://github.com/sherpaservices/Public-Documentation/blob/PublicationsRouter/api/v2/Packaging.md) documentation.
+        --FulltextBoundary--
 
-#### Minimum Metadata + Package request
+If you are carrying out this request you MUST include the **content.packaging_format** field in the notification metadata and populate it with the appropriate format identifier as per the [Packaging Format](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md) documentation.
+
+### 4. Validate Minimum Metadata + Package request
 
 It is possible to send a request with virtually no JSON metadata, instead relying on metadata embedded in the binary Package (e.g. in a JATS XML structure).
 
-To do this, send the bare-minimum JSON notification, with only the format identifier of the package included.  For example:
+To do this, send the bare-minimum JSON notification, with only the format identifier of the [package](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md) included.  For example:
 
     POST /validate?api_key=<api_key>
-    Content-Type: multipart/form-data; boundary=FulltextBoundary
-    
-    --FulltextBoundary
-    
-    Content-Disposition: form-data; name="metadata"
-    Content-Type: application/json
-    
-    {
-        "content" : {
-            "packaging_format" : "https://pubsrouter.jisc.ac.uk/FilesAndJATS"
-        },
-    }
-    
-    --FulltextBoundary
+    Header:
+        Content-Type: multipart/form-data; boundary=FulltextBoundary
     
-    Content-Disposition: form-data; name="content"
-    Content-Type: application/zip
-    
-    [Package]
-    
-    --FulltextBoundary--
+    Body:
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="metadata"
+        Content-Type: application/json
+
+        {
+            "content" : {
+                "packaging_format" : "https://pubsrouter.jisc.ac.uk/FilesAndJATS"
+            },
+        }
 
-#### Possible HTTP Responses
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="content"
+        Content-Type: application/zip
+
+        [Package]
+
+        --FulltextBoundary--
+
+### 5. Possible HTTP Responses
```

Comment:
I think that this shouldn't have a number, and also might be better to move to the top of the validation section.  Maybe including a sentence:

"Any of the validation endpoints listed below will return one of these responses."